docs/en/setup/plugins/forwarder_nativemeter-grpc-forwarder.md

```diff
@@ -6,4 +6,6 @@ This is a synchronization meter grpc forwarder with the SkyWalking meter protoco
 ## Configuration
 |Name|Type|Description|
 |----|----|-----------|
+| upstream_cache_size | int | The upstream cache max size |
+| upstream_cache_second | int | The upstream cache time(second) on each service instance |
```

Comment:
What is `cache time`? I think you mean `upstream_cache_ttl_last_used`?

And what happens if `upstream_cache_size` is reached, but no data could be removed by `upstream_cache_ttl_last_used` policy?

Reply:
> What is cache time? I think you mean upstream_cache_ttl_last_used?

Yes, this means the maximum survival time of data when the instance is no longer sending upstream.

> And what happens if upstream_cache_size is reached, but no data could be removed by upstream_cache_ttl_last_used policy?

It's a kind of LRU Map, if the cache size is reached, it will remove the longest unused data.

Reply:
OK, then let's rename both to `upstream_lru_cache_size` and `pstream_lru_cache_ttl`